### ISSUE TITLE
"K2 Stats (admin)" widget crashes Joomla 4

### ISSUE THREAD

**Issue by cmirnow** [eworkers/K2ForJ4#28]
Enabling widget K2 Stats (admin) in the admin panel of Joomla 4 initiates an error:

```
An error has occurred.
0 Joomla\CMS\HTML\Helpers\Bootstrap::startTabSet(): Return value must be of type string, none returned
```
or

`The server returned a "500 - Whoops, looks like something went wrong."`

Reproduce:
Admin Panel (Joomla 4.2.5, php 8.1) -> Add module to the dashboard -> Select a Module Type -> K2 Stats (admin). Publish the module (position Atum -> "cpanel" or "status").

**Comment by eworkers** [comment]
For 2 or more K2stats published? or for all cases?

**Comment by cmirnow** [comment]
That's a good question... I'm sorry, it's hard for me to answer. Probably the first case. But I'm not sure.

I installed K2 v.2.10.3 on Joomla 3, then upgraded K2 to K2ForJ4, then upgraded Joomla and raised php to 8.0. K2 Stats disappeared from the admin panel. I republished the module and got this error. @eworkers, if this issue is critical, I can give a more confident answer, but that will be late tonight.
